forte/processors/data_augment/algorithms/back_translation_op.py

```diff
@@ -36,22 +37,21 @@ class BackTranslationOp(TextReplacementOp):
     back to the original language, with pretrained
     machine-translation models.
 
-    Args:
-        configs (Config): The configuration for back translation model.
-            It should have the following fields:
+    It will sample from a Bernoulli distribution to decide
+    whether to replace the input, with `prob` as the probability
+    of replacement.
+
+    The configuration should have the following fields:
 
-            - ``'src_lang'``:
-                The source language of back translation.
-            - ``'tgt_lang'``:
-                The target language of back translation.
-            - ``'model_to'``:
-                The full qualified name of the model from source language
-                to target language.
-            - ``'model_back'``:
-                The full qualified name of the model from target language
-                to source language.
-            - ``'device'``:
-                "cpu" for the CPU or "cuda" for GPU.
+    Args:
```

Comment:
I think you have the default value for these, can you mention them here/

Reply:
We actually don't have a default value for the config of the replacement ops.

Reply:
In that case, please add check to make sure the config is valid.